index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
- Add a description on your Pull request about your features implemented. 
- Provide a general information about your project, License, how to use your project install it locally on your README file.